Tools/ssl/multissltests.py

```diff
@@ -410,6 +410,10 @@ def _post_install_300(self):
             ["make", "-j1", "install_ssldirs", "install_fips"],
             cwd=self.build_dir
         )
+        if not os.path.isdir(self.lib_dir):
+            # 3.0.0-beta2 uses lib64 on 64 bit platforms
+            lib64 = self.lib_dir + "64"
+            os.symlink(lib64, self.lib_dir)
```

Comment:
Seems like this change causes all SSL tests to fail on 3.9?

Reply:
That seems unlikely. The ``_post_install_300`` function is only executed ``if self.version.startswith("3.0")``.

Reply:
I have bisected the failing tests in 3.9. Tests are passing in commit for tag v3.9.7. Current 3.9 is failing. The issue was introduced in c967bd523caabb05bf5988449487d7c1717f3ac6 (GH-28153).

@serhiy-storchaka please take a look

Reply:
Thank you @tiran. Shame on me. I have no idea how could I miss these warnings.

#28236 should fix it.

Reply:
I guess you didn't notice because the tests were skipped. Additional OpenSSL tests are only executed for some changes:

https://github.com/python/cpython/blob/c11956a8bddd75f02ccc7b4da7e4d8123e1f3c5f/.github/workflows/build.yml#L53